flink-table/flink-table-planner/pom.xml

```diff
@@ -105,23 +107,15 @@ under the License.
 			</exclusions>
 		</dependency>
 
-		<dependency>
-			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-table-runtime</artifactId>
-			<version>${project.version}</version>
-		</dependency>
+		<!-- Table Runtime (not included in the uber) -->
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-scala_${scala.binary.version}</artifactId>
+			<artifactId>flink-table-runtime</artifactId>
```

Comment:
`provided`?

Reply:
No, as this is a transitive dependency to run the planner. The reason to remove provided is to simplify the setup for who's depending now on table-planner directly (e.g. connector tests). This won't be included in the fat jars anyway.